modules/cms-kit/src/Volo.CmsKit.Admin.Web/Pages/CmsKit/Shared/Components/Comments/CommentSettingViewComponent.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Volo.Abp.AspNetCore.Mvc.UI.Widgets;
+using Volo.Abp.AspNetCore.Mvc;
+
+namespace Volo.CmsKit.Admin.Web.Pages.CmsKit.Shared.Components.Comments;
+
+[Widget(
+    ScriptTypes = new[] { typeof(CommentSettingScriptBundleContributor) },
+    AutoInitialize = true
+)]
+[ViewComponent(Name = "CmsCommentSetting")]
+public class CommentSettingViewComponent : AbpViewComponent
+{
+    public virtual IViewComponentResult Invoke()
+    {
+        return View("~/Pages/CmsKit/Shared/Components/Comments/Index.cshtml");
```

Comment:
Instead of `Index.cshtml`, we name as `Default.cshtml` as I remember. Please check some other components to verify that.